README.md

```diff
@@ -77,6 +77,8 @@ via [Open Collective](https://opencollective.com/mui-org) or via [Patreon](http
   <a href="https://megafamous.com/?utm_source=MUI&utm_medium=referral&utm_content=readme" rel="noopener sponsored" target="_blank"><img height="96" width="96" src="https://mui.com/static/sponsors/megafamous.png" alt="megafamous.com" title="MegaFamous: Buy Instagram followers and likes." loading="lazy" /></a>
   <a href="https://www.dialmycalls.com/?utm_source=MUI&utm_medium=referral&utm_content=readme" rel="noopener sponsored" target="_blank"><img height="96" width="96" src="https://images.opencollective.com/dialmycalls/f5ae9ab/avatar/288.png" alt="dialmycalls.com" title="DialMyCalls: Send text messages, calls, and emails." loading="lazy" /></a>
   <a href="https://goread.io/?utm_source=MUI&utm_medium=referral&utm_content=readme" rel="noopener sponsored" target="_blank"><img height="96" width="96" src="https://images.opencollective.com/goread_io/eb6337d/logo/288.png" alt="goread.io" title="Goread.io: Instagram followers, likes, views, and comments." loading="lazy" /></a>
+  <a href="https://buzzoid.com/buy-instagram-followers/" rel="noopener sponsored" target="_blank"><img height="96" width="96" src="/static/sponsors/buzzoid-square.svg" alt="buzzoid.com" title="Buzzoid: Instant delivery Instagram followers" loading="lazy" /></a>
+  <a href="https://twicsy.com/buy-instagram-followers" rel="noopener sponsored" target="_blank"><img height="96" width="96" src="/static/sponsors/twicsy-square.svg" alt="twicsy.com" title="Twicsy: Instant delivery Instagram followers" loading="lazy" /></a>
```

Comment:
I'm confused, why add Twicsy? They are not active sponsors: https://docs.google.com/spreadsheets/d/1JYYhZN2JDaSXYsDYYyUSYU5pwtJf5hfoa8yJhHTL5ZU/edit?gid=0#gid=0 or https://opencollective.com/twicsy/transactions?searchTerm=MUI.

```suggestion
```

Reply:
I assumed they'd sponsor again since they have before; I'll check with Derek now, if he says yes then we might as well kill two birds with one stone (faster than doing another PR)

Reply:
+1 to batch

Reply:
Per https://app.frontapp.com/open/msg_29a3luvq, we'll be listing Twicsy for 5 months using previous sponsorship as credit. Follow-up will be scheduled for end of January to check back in on continued sponsorship.